### ISSUE TITLE
Navigation menu items disappears during/after upgrade

### ISSUE THREAD

**Issue by jahali** [Puzzlepart/prosjektportalen#479]
Thank you for reporting an issue, suggesting an enhancement, or asking a question. We appreciate your feedback - to help the team understand your needs please complete the below template to ensure we have the details to help. Thanks!

#### Category

[ ] Enhancement

[X] Bug

[ ] Question

#### Environment

[X ] SharePoint Online

[ ] SharePoint 2013

[ ] SharePoint 2016

#### Browser

[ X] Chrome

[X ] Internet Explorer

[ ] Edge

[ ] Safari

[ ] Other

#### Summary

In our installation we have added a link in the navigation menu on Portfolio level.
The link disappears  from time to time.

#### Version

Please specify what version you are using: [ ]

#### Expected / Desired Behavior / Question
We expect our manually added link to be permanent.

#### Observed Behavior
From time to time the link is removed from the navigation.

#### Steps to Reproduce

![image](https://user-images.githubusercontent.com/33092566/35115076-41b31b82-fc87-11e7-8207-40ee602ca426.png)

**Comment by olemp** [comment]
This might be happening during upgrading @jahali. We'll have a look at the upgrade script.

**Comment by jahali** [comment]
We have just upgraded our installation to version: [ v2.1.60.4dea91b3]
Even in this upgrade we got the same problem.
Upgrade.ps1 completely overwrites the left side navigation, wiping out any custom navigation items like Transmittals.unrelated to Project Portal.
We could not find any key, which would allow preventing this removal. 
So the custom items should needs to be manually restored in the navigation after the upgrade.

Is it possible to add a key to prevent this.
On the same basis as the key "-SkipCustomActionRemoval" does?
